### PR TITLE
CRUK Blog post

### PR DESCRIPTION
**Pull request by elpabl0**
crud blog post

### CODE REVIEW

File: _posts/2016-04-19-cruk-data-hack.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: The price of light is less than the cost of darkness
+author: Emma Nash and Aisling Foley
```
Comment (emmanash) [inline _posts/2016-04-19-cruk-data-hack.md]: You should add your name ^_^

**Comment by dave-hillier** [comment]
The image looks like the cells arent quite right. We should clean it up.

File: _posts/2016-04-19-cruk-data-hack.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* Participation
+* Self-empowerment
+* Improved or new private products and services
+* Innovation
+* Improved efficiency of government services
+* Improved effectiveness of government services
+* Impact measurement of policies
+* New knowledge from combined data sources and patterns in large data volumes
+
+Specifically for Cancer Research UK, reducing the effort required to produce useful profiles and infographics is of benefit to the organisation - increasing the freshness of data available, whilst reducing the effort required to share it in a useful way. By creating an API on top of the existing data, profiles and infographics can be automatically generated and updated with the most recent information, without having the re-engage designers.
```
Comment (emmanash) [inline _posts/2016-04-19-cruk-data-hack.md]: At the end of this line it says "without having the re-engage designers" - I can't figure out what you're trying to say here?

Comment (elpabl0) [inline _posts/2016-04-19-cruk-data-hack.md]: Fixed it.

Comment (elpabl0) [inline _posts/2016-04-19-cruk-data-hack.md]: @dave-hillier Does it look the same on your screen? Might have to live with it....

**Pull request merged**